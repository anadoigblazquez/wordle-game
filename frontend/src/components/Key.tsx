
type Letter = {
  keyLetter: string
}

const Key = ({ keyLetter }: Letter) => {
  return (
    <div className="
      font-bold
      text-xl 
      rounded-md 
      bg-gray-300
      min-w-[40px]
      px-2 
      h-[55px]
      flex
      justify-center
      flex-wrap
      content-center
      cursor-pointer">
        {keyLetter}
      </div>
  )
}

export default Key