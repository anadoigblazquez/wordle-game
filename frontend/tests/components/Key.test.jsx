import { render, screen } from "@testing-library/react";
import Key from '../../src/components/Key';

describe('Tests on Key component', () =>  {
    const className = 'className';

    test('should render Key', () => {
        const { container } = render( <Key className={ className } /> );
        expect( container ).toMatchSnapshot();
    })
})