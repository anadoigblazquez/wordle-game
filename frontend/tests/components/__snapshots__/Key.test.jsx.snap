// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on Key component should render Key 1`] = `
<div>
  <div
    class=" font-bold text-xl  rounded-md  bg-gray-300 min-w-[40px] px-2  h-[55px] flex justify-center flex-wrap content-center cursor-pointer"
  />
</div>
`;
